src/Orbits.jl

```diff
@@ -1,194 +1,385 @@
 using LinearAlgebra
-using Symbolics
-using IfElse
+
+###########################################################
+### 
+### Standard orbital parameter calculations and conversions
+### 
+###########################################################
 
 """
-    kepler_a_from_P(P,m1,m2)
+    kepler_a_from_P(;m_1, m_2, P)
 
 Obtain semimajor axis from period using Kepler's third law
 
-#Arguments:
+# Arguments:
+- m_1: mass of first companion [Msun]
+- m_2: mass of 2nd companion   [Msun]
 - P:  orbital period           [d]
-- m1: mass of first companion [Msun]
-- m2: mass of 2nd companion   [Msun]
 
-#Output:
+# Output:
 - a: semi-major axis of the orbit [cm]
 """
-function kepler_a_from_P(P,m1,m2)
-    return cbrt((P*day_in_sec)^2*cgrav*(m1+m2)*m_sun/(4.0*π^2))
+function kepler_a_from_P(;m_1, m_2, P)
+    return cbrt((P*day_in_sec)^2*cgrav*(m_1+m_2)*m_sun/(4.0*π^2))
 end
 
-
 """
-    kepler_P_from_a(a,m1,m2)
+    kepler_P_from_a(;m_1, m_2, a)
 
 Obtain period from semimajor axis using Kepler's third law
 
-#Arguments:
-- a:  semi-major axis of the orbit [cm]
-- m1: mass of first companion     [Msun]
-- m2: mass of 2nd companion       [Msun]
+# Arguments:
+- m_1: mass of first companion      [Msun]
+- m_2: mass of 2nd companion        [Msun]
+- a:  semi-major axis of the orbit  [cm]
 
-#Output:
+# Output:
 - P: the orbital period            [d]
 """
-function kepler_P_from_a(a,m1,m2)
-    return sqrt(4.0*π^2*a^3/(cgrav*(m1+m2)*m_sun))/(day_in_sec)
+function kepler_P_from_a(;m_1, m_2, a)
+    return sqrt(4.0*π^2*a^3/(cgrav*(m_1+m_2)*m_sun))/(day_in_sec)
+end
+
+
+"""
+    average_orbital_velocity(;m_1, m_2, a)
+
+Calculate the average orbital velocity, or equivalently
+the fixed orbital velocity for a circular orbit.
+
+# Arguments:
+- m_1: mass of first companion        [Msun]
+- m_2: mass of 2nd companion          [Msun]
+- a:  semi-major axis of the orbit    [cm]
+
+# Output:
+- v_orb: the average orbital velocity [km/s]
+"""
+function average_orbital_velocity(;m_1, m_2, a)
+    return sqrt(cgrav*(m_1+m_2)*m_sun/ a)/(km) 
```

Comment:
I'm not sure I understand what would be the orbital velocity. What this is is the relative orbital velocity between the two components. I am also not sure if this corresponds to any meaningful average.

Reply:
I had thought this was an average over the orbit, no? At any rate I've renamed it for just the circular functions.

Reply:
no, this is the velocity of one star relative to the other one. Constant for a circular orbit, variable for the elliptical one as discussed in the overleaf